NMS:13250: Proofread Basic Horizon Deployment (#3437)
OpenNMS/opennms: docs/modules/operation/pages/webui/dashboard.adoc

[[ga-dashboard-resource-graph-viewer]]
=== Resource Graph Viewer

> To give a quick entry point diagnose performance issues a Resource Graph Viewer lets you navigate to time series data reports which are filtered in the context of the Surveillance View.

Reviewer: ```suggestion
The Resource Graph Viewer lets you navigate time series data reports to give a quick entry point for diagnosing performance issues.
This viewer filters nodes based on the Surveillance View context and shows one graph report at a time.
```

.Show time series based performance with the Resource Graph Viewer
image::dashboard/06_dashboard-resource-graphs.png[]

> It lets you navigate sequentially through resource graphs provided by nodes filtered by the Surveillance View context and selection and shows one graph report at a time.

Reviewer: Merge with text above the image.
```suggestion
```

[[ga-dashboard-advanced-configuration]]
== Advanced configuration

> The Surveillance View component lets you to model multiple views for different monitoring contexts.

Reviewer: ```suggestion
The Surveillance View component lets you model multiple views for different monitoring contexts.
```
